src/test/java/de/tum/cit/aet/artemis/lecture/AttachmentVideoUnitsIntegrationTest.java

```diff
@@ -106,7 +106,7 @@ void testAll_LectureWithoutCourse_shouldReturnBadRequest() throws Exception {
         request.get("/api/lecture/lectures/" + invalidLecture.getId() + "/attachment-video-units/slides-to-remove/any-file", HttpStatus.BAD_REQUEST,
                 LectureUnitSplitInformationDTO.class, params);
         request.postListWithResponseBody("/api/lecture/lectures/" + invalidLecture.getId() + "/attachment-video-units/split/any-file", lectureUnitSplits, AttachmentVideoUnit.class,
-                HttpStatus.BAD_REQUEST);
+                HttpStatus.FORBIDDEN);
```

Comment:
Shouldn't this stay a bad request? 

Might this be related to one of the adjusted authorization checks which would mean that the test user in this test would just be needed to be added to the lecture (or course)?